spec/models/rule_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Rule, type: :model do
+  it { is_expected.to validate_presence_of(:code) }
+  it { is_expected.to validate_uniqueness_of(:code) }
+  it { is_expected.to validate_presence_of(:description) }
+
+  it "the active should be true by default" do
+    rule = create(:rule)
```

Comment:
you either use the `let` or not. If you are going to use `let` in some them you better use in all cases